Asaas/grails-app/jobs/asaas/ExpiredPaymentJob.groovy

```diff
@@ -0,0 +1,21 @@
+package asaas
+
+import asaas.Payment
+import asaas.PaymentService
+
+import java.time.LocalDate
+import java.time.ZoneId
+import java.util.Date
+
+class ExpiredPaymentJob {
+
+    PaymentService paymentService
+
+    static triggers = {
+        cron name:   "expiredPaymentTrigger",  cronExpression: "0 0 0 * * ?"
```

Comment:
```suggestion
        cron name: "expiredPaymentTrigger", cronExpression: "0 0 0 * * ?"
```
Cuida com espaços excessivos
Coloca uma janela de execução, exemplo:

Todo dia, iniciando as 0h até 2h, executando a cada 10 minutos
Assim se o servidor tiver algum problema, ele for corrigido, as chances da job nao ser executada naquele dia, diminuem. Execução única é polêmico demais